### ISSUE TITLE
[Accessibility Visual Requirement] - No tooltips for reorder icon in query operation options (MAS ID: 260681b))

### ISSUE THREAD

**Issue by ckbedwell** [grafana/grafana#72841]
**Steps to reproduce**:
Go to an explore page with a Query. Hover or focus the reorder icon. Notice no tooltip to indicate its purpose.

**Actual Result**:
Tooltip is not defined for reorder icon in the query operation actions.

**Expected Result**
 Tooltip should be defined for reorder icon in query operation actions.
 
**Relevant WCAG Criteria:**
Could possibly make an argument for [3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.1#labels-or-instructions)?

![image](https://github.com/grafana/grafana/assets/6906380/4066cb5f-7d53-4126-a0f3-ea4a7dbbafaa)

**Component**
`<QueryOperationRow />`
